staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go

```diff
@@ -57,6 +57,10 @@ func (ctx *ChannelContext) Cleanup(module string) {
 
 // Send send msg to a module. Todo: do not stuck
 func (ctx *ChannelContext) Send(module string, message model.Message) {
+	// check if msg is sync
+	if message.Header.Sync {
+		klog.Warningf("Get a sync-msg when use beehive.Send(),msg.Header:{%v},msg.Route:{%v}",message.Header,message.Router)
```

Comment:
In fact it would be useful for debug, i prefer combine them into one debug level log.

Reply:
yes it is, and how to? :)

Reply:
By mistake, i mean change to debug level instead.

Reply:
but there is no `klog.Debug`

Reply:
or klog.V(4).Infof()? as https://www.cnblogs.com/iiiiher/p/7879817.html